$NetBSD: patch-aa,v 1.3 2020/06/21 04:09:24 mef Exp $

date: 2007-01-31 08:52:53 +0900;  author: wiz;  state: Exp;
Second try at fixing the bulk build of this package --
let the curses test always succeed.
Cross-compiling: assume curses without the raw() bug and a working realloc.

--- configure.orig
+++ configure
@@ -3259,10 +3259,7 @@
 ORIG_LIBS=$LIBS
 LIBS="$LIBS -lcurses"
 if test "$cross_compiling" = yes; then :
-  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error "cannot run test program while cross compiling
-See \`config.log' for more details." "$LINENO" 5; }
+  curses_ok=yes	# assumed: cross-compiling (pkgsrc curses)
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
@@ -3294,6 +3291,8 @@
 fi
 
 LIBS=$ORIG_LIBS
+curses_ok=yes
+
 if test "$curses_ok" = yes; then
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking for modern curses without raw() bug" >&5
 $as_echo_n "checking for modern curses without raw() bug... " >&6; }
@@ -3843,10 +3842,8 @@
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking for broken realloc" >&5
 $as_echo_n "checking for broken realloc... " >&6; }
 if test "$cross_compiling" = yes; then :
-  { { $as_echo "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-$as_echo "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error "cannot run test program while cross compiling
-See \`config.log' for more details." "$LINENO" 5; }
+  { $as_echo "$as_me:${as_lineno-$LINENO}: result: no (assumed: cross-compiling)" >&5
+$as_echo "no (assumed: cross-compiling)" >&6; }
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
